nixos/modules/services/x11/desktop-managers/cinnamon.nix

```diff
@@ -220,13 +220,6 @@ in
       programs.bash.vteIntegration = mkDefault true;
       programs.zsh.vteIntegration = mkDefault true;
 
-      # Qt application style
-      qt = {
-        enable = mkDefault true;
```

Comment:
Not sure if `qt.enable = true` is still required or not, as we add Qt 6 wayland plugin in this way.